packages/nvlsm/infiniband-guid.service

```diff
@@ -0,0 +1,13 @@
+
+[Unit]
+Description=Detect Infiniband devices and write the GUID env file if present
+RequiresMountsFor=/etc
```

Comment:
You shouldn't need these `RequiresMountsFor` rules for a unit with default dependencies.
```suggestion
```